lib/folio_record.rb

```diff
@@ -87,6 +87,19 @@ def sirsi_holdings
       _current_library, current_location = LocationsMap.for(item.dig('location', 'temporaryLocation', 'code'))
       current_location ||= Folio::StatusCurrentLocation.new(item).current_location
 
+      # TODO: we don't handle multiple courses for a single item, because it's beyond parity
+      item_courses = courses.select { |course| course[:listing_id] == item['courseListingId'] }
+      course = item_courses.first
+
+      # We use loan types as loan periods for course reserves so that we don't need to check circ rules
+      # Items on reserve in FOLIO usually have a temporary loan type that indicates the loan period
+      # "3-day reserve" -> "3-day loan"
+      course_reserves = {
```

Comment:
Can we remove the conditional from line 114 and define `course_reserves = {}` just prior to this line?

Reply:
Done